src/Test/WebDrapo.Test/Pages/Format12Hour.Test.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml"><head>
```

Comment:
We already have a unit test for dates in Format.html. Can you include the test over there ? 
Take care when changing the API ~/Data/GetDates. We could have more places do adjust